cmd/mapt/cmd/azure/hosts/rhel.go

```diff
@@ -0,0 +1,143 @@
+	"github.com/redhat-developer/mapt/pkg/provider/util/instancetypes"
+	"github.com/redhat-developer/mapt/pkg/util"
+
+	spotAzure "github.com/redhat-developer/mapt/pkg/spot/azure"
+	"github.com/redhat-developer/mapt/pkg/util/ghactions"
+	"github.com/redhat-developer/mapt/pkg/util/logging"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+const (
+	cmdRHEL     = "rhel"
+	cmdRHELDesc = "RHEL operations"
+)
+
+func GetRHELCmd() *cobra.Command {
+	c := &cobra.Command{
+		Use:   cmdRHEL,
+		Short: cmdRHELDesc,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := viper.BindPFlags(cmd.Flags()); err != nil {
+				return err
+			}
+			return nil
+		},
+	}
+	c.AddCommand(getCreateRHEL(), getDestroyRHEL())
+	return c
+}
+
+func getCreateRHEL() *cobra.Command {
+	c := &cobra.Command{
+		Use:   params.CreateCmdName,
+		Short: params.CreateCmdName,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := viper.BindPFlags(cmd.Flags()); err != nil {
+				return err
+			}
+			// Initialize context
+			maptContext.Init(
+				viper.GetString(params.ProjectName),
+				viper.GetString(params.BackedURL),
+				viper.GetString(params.ConnectionDetailsOutput),
+				viper.GetStringMapString(params.Tags))
+
+			// ParseEvictionRate
+			var spotToleranceValue = spotAzure.DefaultEvictionRate
+			if viper.IsSet(paramSpotTolerance) {
+				var ok bool
+				spotToleranceValue, ok = spotAzure.ParseEvictionRate(
+					viper.GetString(paramSpotTolerance))
+				if !ok {
+					return fmt.Errorf("%s is not a valid spot tolerance value", viper.GetString(azparams.ParamSpotTolerance))
+				}
+			}
+			instanceRequest := &instancetypes.AzureInstanceRequest{
+				CPUs:       viper.GetInt32(params.CPUs),
+				MemoryGib:  viper.GetInt32(params.Memory),
+				Arch:       util.If(viper.GetString(params.LinuxArch) == "arm64", instancetypes.Arm64, instancetypes.Amd64),
+				NestedVirt: viper.GetBool(params.NestedVirt),
+			}
+
+			// Initialize gh actions runner if needed
+			if viper.IsSet(params.InstallGHActionsRunner) {
+				err := ghactions.InitGHRunnerArgs(viper.GetString(params.GHActionsRunnerToken),
+					viper.GetString(params.GHActionsRunnerName),
+					viper.GetString(params.GHActionsRunnerRepo),
+					viper.GetStringSlice(params.GHActionsRunnerLabels))
+				if err != nil {
+					logging.Error(err)
+				}
+			}
+
+			if err := azureRHEL.Create(
+				&azureRHEL.Request{
+					Prefix:               viper.GetString(params.ProjectName),
+					Location:             viper.GetString(paramLocation),
+					VMSizes:              viper.GetStringSlice(paramVMSize),
+					InstanceRequest:      instanceRequest,
+					Version:              viper.GetString(paramLinuxVersion),
+					Arch:                 viper.GetString(params.LinuxArch),
+					SubsUsername:         viper.GetString(params.SubsUsername),
+					SubsUserpass:         viper.GetString(params.SubsUserpass),
+					ProfileSNC:           viper.IsSet(params.ProfileSNC),
```

Comment:
You mean this one?

Reply:
i meant for the `NestedVirt` on line 70 it should be `true` also in case of `ProfileSNC`, 

so `NestedVirt: viper.GetBool(params.NestedVirt) || viper.GetBool(params.ProfileSNC)`

Reply:
ohhh yeah totally right was not thinking on that